### PR TITLE
TOOLS-2461 upgrade Manta/Triton 19.[12] components to 19.4

### PR DESCRIPTION
**Pull request by timfoster**

### CODE REVIEW

**Comment by timfoster** [comment]
I've added test results to the ticket, showing us running the mantav1 mako tests on a coal instance running the mantav1 manta-mako 19.4 image.

File: package.json [hunk 1]
```diff
@@ -1,18 +1,19 @@
...
         "license": "MPL-2.0",
         "private": true,
         "dependencies": {
-                "assert-plus": "0.1.1",
-                "bunyan": "0.21.4",
+                "assert-plus": "1.0.0",
+                "bunyan": "1.8.12",
                 "carrier": "0.1.7",
-                "manta": "2.0.7",
-                "moray": "git+https://github.com/joyent/node-moray.git#fd5781bc25a9bfe2ba82167664639753fb9f0ca5",
```
Comment (trentm) [inline package.json:13]: Doesn't that break the bin/mako_rebalance.js script? Granted that may never again be used. ... this is in the class of those things that motivated dropping crap from mantav2.

Other than mako's tests, I think that is the only thing here using the sdcnode that ships with the mako image. I believe minnow has its own.  Was minnow working in your test?

Comment (timfoster) [reply package.json:13]: Minnow is working - heartbeating to moray, reflecting actual space on /manta and dealing with 1.moray.earth.example.com being rebooted. It's using its own prebuilt-node-v0.10.48-zone-fd2cc906-8938-11e3-beab-4359c665ac99 (1.6.3 sdcnode)

You're right that this breaks the mako_rebalance script :/  Maybe dropping moray back in (at the same version as minnow is using, 3.0.7) will fix it, but I'll spend time making sure I can get that rebalancing working in the first place so I know whether I've made it better or worse.

**Comment by timfoster** [comment]
The rebalancer now runs, however in my test coal instance, it doesn't seem to be rebalancing objects as expected. That said, I suspect this is pilot error on my part: I had identical behaviour using the stock pkgsrc 2018Q1 manta-mako image without these changes.

I'd like to not spend too much more time on supporting mantav1, given that the rebalancer has been totally rewritten for mantav2, so this is effectively dead code I'm trying to beat into life..

File: package.json [hunk 2]
```diff
@@ -1,18 +1,20 @@
...
+                "manta": "5.2.1",
+                "moray": "3.7.0",
                 "posix-getopt": "1.0.0",
-                "vasync": "1.6.4"
+                "restify": "4.3.4",
+                "triton-metrics": "1.0.2",
```
Comment (trentm) [inline package.json:16]: Why are restify and triton-metrics being added? I don't see what is using those in manta-mako.git

Comment (timfoster) [reply package.json:16]: Sorry, that looks like stuff we only needed in ./bin/garbage-deleter.js in the master (mantav2) branch. I'll remove these.

**Pull request merged**